### PR TITLE
Add Windowing samples

### PR DESCRIPTION
**Pull request by samanthamsong**
Adding windowing sample projects:
1) cs_winui
2) cpp_winui

### CODE REVIEW

File: Samples/Windowing/cs_winui/SampleApp/SampleApp.csproj [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <None Remove="PresenterPage.xaml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-experimental1" />
+    <Manifest Include="$(ApplicationManifest)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.19041.18" />
```
Comment (angelazhangmsft) [inline Samples/Windowing/cs_winui/SampleApp/SampleApp.csproj:23]: We should remove the FrameworkReference override - this was a temporary workaround before this version of .NET SDK was released

**Comment by angelazhangmsft** [comment]
nit: rename to cs-winui/cpp-winui for consistency with other samples :)

File: Samples/Windowing/cpp_winui/SampleApp (Package)/Package.appxmanifest [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <Identity
+    Name="7089e2db-be6f-4635-9abc-c1430d700d1a"
+    Publisher="CN=rokarman"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>SampleApp (Package)</DisplayName>
+    <PublisherDisplayName>rokarman</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
```
Comment (rkarman) [inline Samples/Windowing/cpp_winui/SampleApp (Package)/Package.appxmanifest:18]: Should probably not use my identity for the package. :) (goes for both projects)

File: Samples/Windowing/cs-winui/SampleApp/WindowingInterop.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+using Microsoft.UI;
+using Microsoft.UI.Windowing;
+using WinRT;
+
+namespace SampleApp
+{
+    [ComImport]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB")]
+    internal interface IWindowNative
```
Comment (angelazhangmsft) [inline Samples/Windowing/cs-winui/SampleApp/WindowingInterop.cs:12]: We should use the new COM interop APIs here: https://docs.microsoft.com/windows/apps/desktop/modernize/winrt-com-interop-csharp

**Review by angelazhangmsft — APPROVED** [review]

**Pull request merged**